src/api/Nhs.Appointments.Api/Functions/GetSitesByAreaFunction.cs

```diff
@@ -41,6 +41,18 @@ public class GetSitesByAreaFunction(
     [OpenApiParameter("accessNeeds", In = ParameterLocation.Query, Required = false, Type = typeof(string[]),
         CollectionDelimiter = OpenApiParameterCollectionDelimiterType.Comma,
         Description = "Required access needs used to filter sites")]
+    [OpenApiParameter("services", In = ParameterLocation.Query, Required = false, Type = typeof(string[]),
+        CollectionDelimiter = OpenApiParameterCollectionDelimiterType.Comma,
+        Description =
+            "Optional parameter to be used alongside from and until. When all three of these parameters are used, it filters the results to only those sites that support the services within that date range")]
+    [OpenApiParameter("from", In = ParameterLocation.Query, Required = false, Type = typeof(string),
+        CollectionDelimiter = OpenApiParameterCollectionDelimiterType.Comma,
```

Comment:
I don't think you need the CollectionDelimiter lines for the from/until dates as they're just strings